sdk/samples/iot/README.md

```diff
@@ -854,6 +854,15 @@ This section provides an overview of the different samples available to run and
 
   This [sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/paho_iot_provisioning_sas_sample.c) registers a device with the Azure IoT Device Provisioning Service. It will wait to receive the registration status before disconnecting. SAS authentication is used.
 
+### IoT Hub ECC Server Certificate Chain
```

Comment:
This falls under ## Sample Descriptions currently.  Did you intend that, or would you prefer it be a section up?
```suggestion
## IoT Hub ECC Server Certificate Chain
```